src/modules/file/file.usecase.ts

```diff
@@ -32,11 +32,24 @@ export class FileUseCases {
     private fileRepository: SequelizeFileRepository,
     @Inject(forwardRef(() => ShareUseCases))
     private shareUseCases: ShareUseCases,
+    @Inject(forwardRef(() => FolderUseCases))
     private folderUsecases: FolderUseCases,
     private network: BridgeService,
     private cryptoService: CryptoService,
   ) {}
 
+  getByUserExceptParents(arg: any): Promise<File[]> {
+    throw new Error('Method not implemented.');
+  }
+
+  getByFileIdAndUser(arg: any): Promise<File> {
+    throw new Error('Method not implemented.');
+  }
+
+  async deleteFilePermanently(file: File, userMock: User): Promise<void> {
```

Comment:
The user is not a mock on the usecase, so I will call it user instead of userMock